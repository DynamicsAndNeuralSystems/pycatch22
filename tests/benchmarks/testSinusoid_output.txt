0.13687262547490, DN_OutlierInclude_n_001_mdrmd, 8.703000
-0.13777244551090, DN_OutlierInclude_p_001_mdrmd, 6.593000
1.27292807660985, DN_HistogramMode_5, 0.041000
-1.12758575737747, DN_HistogramMode_10, 0.039000
1.64629633438112, CO_Embed2_Dist_tau_d_expfit_meandiff, 8.099000
119.74079600337870, CO_f1ecac, 6.009000
2.00000000000000, CO_FirstMin_ac, 4.579000
1.07949995567938, CO_HistogramAMI_even_2_5, 0.320000
-0.00017509308632, CO_trev_1_num, 0.110000
0.00625000000000, FC_LocalSimple_mean1_tauresrat, 13.819000
0.14051134209844, FC_LocalSimple_mean3_stderr, 0.190000
1.00000000000000, IN_AutoMutualInfoStats_40_gaussian_fmmi, 1.845000
0.82200000000000, MD_hrv_classic_pnn40, 0.041000
6.00000000000000, SB_BinaryStats_diff_longstretch0, 0.051000
301.00000000000000, SB_BinaryStats_mean_longstretch1, 0.046000
1.38225822030023, SB_MotifThree_quantile_hh, 1.417000
0.77551020408163, SC_FluctAnal_2_rsrangefit_50_1_logi_prop_r1, 3.221000
0.69387755102041, SC_FluctAnal_2_dfa_50_1_2_logi_prop_r1, 1.645000
0.98817221358274, SP_Summaries_welch_rect_area_5_1, 2.410000
0.00997087512126, SP_Summaries_welch_rect_centroid, 2.218000
0.01005896635449, SB_TransitionMatrix_3ac_sumdiagcov, 6.992000
628.00000000000000, PD_PeriodicityWang_th0_01, 17.879000

