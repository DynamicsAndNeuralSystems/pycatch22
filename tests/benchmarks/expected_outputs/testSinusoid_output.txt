DN_HistogramMode_5, 1.272928076609852
DN_HistogramMode_10, -1.1275857573774708
CO_f1ecac, 119.7407960033787
CO_FirstMin_ac, 2
CO_HistogramAMI_even_2_5, 1.0794999556793812
CO_trev_1_num, -0.00017509308632170917
MD_hrv_classic_pnn40, 0.822
SB_BinaryStats_mean_longstretch1, 301.0
SB_TransitionMatrix_3ac_sumdiagcov, 0.010058966354491846
PD_PeriodicityWang_th0_01, 628
CO_Embed2_Dist_tau_d_expfit_meandiff, 1.646296334381124
IN_AutoMutualInfoStats_40_gaussian_fmmi, 1.0
FC_LocalSimple_mean1_tauresrat, 0.00625
DN_OutlierInclude_p_001_mdrmd, -0.13777244551089785
DN_OutlierInclude_n_001_mdrmd, 0.13687262547490497
SP_Summaries_welch_rect_area_5_1, 0.9881722135827378
SB_BinaryStats_diff_longstretch0, 6.0
SB_MotifThree_quantile_hh, 1.3822582203002276
SC_FluctAnal_2_rsrangefit_50_1_logi_prop_r1, 0.7755102040816326
SC_FluctAnal_2_dfa_50_1_2_logi_prop_r1, 0.6938775510204082
SP_Summaries_welch_rect_centroid, 0.009970875121257324
FC_LocalSimple_mean3_stderr, 0.1405113420984411
DN_Mean, 0.15152112682263552
DN_Spread_Std, 0.7152912579336829
